adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/VariantContextRDD.scala

```diff
@@ -124,7 +102,8 @@ case class VariantContextRDD(rdd: RDD[VariantContext],
    *   valid VCF header. Default is false.
    */
   def saveAsVcf(filePath: String,
-                asSingleFile: Boolean = false) {
+                asSingleFile: Boolean = false,
+                stringency: ValidationStringency = ValidationStringency.LENIENT) {
```

Comment:
Yeah, if you go looking you'll find lots of other missing @param tags for `stringency`, or those that don't document the default value, and the doc strings are not consistent